piptools/sync.py

```diff
@@ -13,6 +13,7 @@
     'pip-review',
     'setuptools',
     'wheel',
+    'pkg-resources',
```

Comment:
Should this be marked as a workaround for an [alleged Ubuntu bug](https://bugs.launchpad.net/ubuntu/+source/python-pip/+bug/1635463) or would ignoring `pkg-resources` be a good idea even independent of that?